recipe/search-machine-id-on-var-lib.patch

```diff
@@ -0,0 +1,18 @@
+--- dbus/dbus-sysdeps-unix.c	2017-02-07 16:18:18.107041873 -0200
++++ dbus/dbus-sysdeps-unix_new.c	2017-02-07 16:58:05.308661710 -0200
+@@ -3866,6 +3866,15 @@
+ 
+   dbus_error_free (error);
+ 
++  /* Fallback to the machine ID of system-wide dbus*/
```

Comment:
This is just a copy of the code above which looks for DBUS_MACHINE_UUID_FILE: https://cgit.freedesktop.org/dbus/dbus/tree/dbus/dbus-sysdeps-unix.c?h=dbus-1.10&id=dbus-1.10.10#n3861